internal/fileutils/fileutils.go

```diff
@@ -17,21 +17,22 @@ import (
 	"time"
 	"unicode"
 
+	"github.com/gofrs/flock"
+	"github.com/thoas/go-funk"
+
 	"github.com/ActiveState/cli/internal/assets"
 	"github.com/ActiveState/cli/internal/errs"
 	"github.com/ActiveState/cli/internal/locale"
 	"github.com/ActiveState/cli/internal/logging"
 	"github.com/ActiveState/cli/internal/multilog"
 	"github.com/ActiveState/cli/internal/rollbar"
-	"github.com/gofrs/flock"
-	"github.com/thoas/go-funk"
 )
 
 // nullByte represents the null-terminator byte
 const nullByte byte = 0
 
 // FileMode is the mode used for created files
-const FileMode = 0644
+const FileMode = 0o644
```

Comment:
This and similar changes are due to the current recommendation that octal notation be more explicit. My linter kept changing this, and I got tired of trying to fix or avoid it.